### PR TITLE
Add workflow to remove CARGO_TARGET_DIR daily if disk usage more than max_used

### PR DESCRIPTION
**Pull request by liya2017**
Add workflow to remove CARGO_TARGET_DIR daily if disk usage more than max_used

Please hold on this PR due to below things:

- [x] 1. Need discuss the max_used value
- [x] 2. The final job status will be failed when the check step exit 1, need find a way to avoid this.
- [x] 3. Need add the unique label for each runner incase this workflow should run on each runner.

### CODE REVIEW

**Comment by nervos-bot-user** [comment]
Can one of the admins verify this patch?

**Comment by nervos-bot-user** [comment]
Can one of the admins verify this patch?

File: .github/workflows/remove-cargo-target-weeklly.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+name: Remove Cargo target weekly
+
+on:
+  schedule:
+    # Every day at 1 am
```
Comment (yangby-cryptape) [inline .github/workflows/remove-cargo-target-weeklly.yml:5]: The title is `name: Remove Cargo target weekly`, but this comment is `# Every day at 1 am`. 😕 

How about merge this into `.github/workflows/remove-old-artifacts.yml`?

**Review by nervos-bot-user — DISMISSED** [review]
A single job will be dispatched to exactly one runner. Thus, only one self-hosted runner was cleaned.

We should deploy this cronjob to all self-hosted runners, but not to workflows.

**Comment by liya2017** [comment]
> * Need add the unique label for each runner incase this workflow should run on each runner.

Need help on this part since I have no access to see all runner & add labels

File: .github/workflows/remove-cargo-target-weeklly.yml (hunk 1 above)
Comment (liya2017) [reply .github/workflows/remove-cargo-target-weeklly.yml]: done

File: .github/workflows/remove-old-artifacts.yml [hunk 2]
```diff
@@ -16,3 +16,64 @@ jobs:
         age: '1 hour'
         skip-tags: true
         skip-recent: 10
+
+  clean-cargo-target:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [Linux-runner,mac-runner,win-runner]
```
Comment (doitian) [inline .github/workflows/remove-old-artifacts.yml:24]:
```suggestion
        os:
          - host-001                                                                                                                                                 
          - host-002                                                                                                                                                 
          - host-003                                                                                                                                                 
          - host-004                                                                                                                                                 
          - host-005                                                                                                                                                 
          - host-006                                                                                                                                                 
          - host-007                                                                                                                                                 
          - host-008                                                                                                                                                 
          - host-009                                                                                                                                                 
          - host-010                                                                                                                                                 
          - host-011
```

**Pull request closed without merging**